pkgs/applications/window-managers/cagebreak/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkg-config, wayland, scdoc, makeWrapper
+, wlroots, wayland-protocols, pixman, libxkbcommon
+, cairo , pango, fontconfig, pandoc, udev
+, xwayland ? null
+, nixosTests
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cagebreak";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "project-repo";
+    repo = "cagebreak";
+    rev = version;
+    sha256 = "+Ww1rsUR7qe/BixLPR8GiRc3C6QmpLzWpT2wym8b4/M=";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config wayland scdoc makeWrapper ];
+
+  buildInputs = [
+    wlroots wayland wayland-protocols pixman libxkbcommon cairo
+    pango.dev fontconfig pandoc udev
+  ];
+
+  outputs = [ "out" "contrib" ];
+
+  mesonFlags = [
+    "-Dxwayland=${stdenv.lib.boolToString (xwayland != null)}"
+    "-Dversion_override=${version}"
+  ];
+
+  mesonBuildType = "release";
```

Comment:
why doesn't the default buildtype not work?

Reply:
As far as I remember the default Meson is precisely "release"...

Reply:
We had it changed to plain https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/meson/setup-hook.sh#L19 in nixos. As most linux distributions.

Reply:
Both `plain` and `release` work. I used the recommended `release` type from the README.